yt/chyt/server/subquery.cpp

```diff
@@ -254,7 +255,7 @@ class TInputFetcher
                     EKeyConditionScale::TopLevelDataSlice,
                     dataSlice->LowerLimit().KeyBound,
                     dataSlice->UpperLimit().KeyBound,
-                    dataSlice->GetInputStreamIndex()).can_be_true;
+                    operandIndex).can_be_true;
```

Comment:
I don't understand why this change is needed

Reply:
This is the consequence of the change in input stream index (it used to reflect the operand index and now it doesn't).